.github/ISSUE_TEMPLATE/questions-help-support.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Questions/Help/Support
+about: Do you have a question or need help to start running elephant?
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Questions and Help**
+
+_Please note that this issue tracker is not a help form and this issue will be closed. Before asking a new question, please check our [NeuralEnsembe forum](https://groups.google.com/forum/#!forum/neuralensemble), where you can discuss any NeuralEnsemble resource, including python-elephant._
```

Comment:
```suggestion
_Please note that this issue tracker is not a help form and this issue will be closed. Before asking a new question, please check our [NeuralEnsembe forum](https://groups.google.com/forum/#!forum/neuralensemble), where you can discuss any NeuralEnsemble resource, including python-elephant._
```
```suggestion
_Please note that this issue tracker is not a help form and this issue may be closed and redirected to a more appropriate communication channel. Before asking a new question, please check our [NeuralEnsembe forum](https://groups.google.com/forum/#!forum/neuralensemble), where you can discuss any NeuralEnsemble resource, including Elephant._
```

Reply:
Regarding the latter point, I found it too brutal to simply state that the issue will be closed, but would rather suggest that the issue would likely be closed and then taken to the appropriate place.

I assume this appropriate place would depend on the question. if we have an answer, we could post the question with the answer on NeuralEnsemble. If not, it may actually turn into a feature request.

Reply:
I think the idea is that a question is NOT an issue and once the question is answered, it's closed rather than keeping it open till the answer is found. I don't know why pytorch makes it so brutal. My guess is that they receive too many questions :)
Anyway, I'll leave your version.

Reply:
Yes, I agree. I was just thinking that in some cases the question may actually turn out to be an real issue without the questioner realizing it :-)